comp/core/log/impl-trace/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/DataDog/datadog-agent/comp/core/log/impl-trace
 
-go 1.21.0
+go 1.22.0
```

Comment:
The old `1.21.0` go version was a mistake, right ?

Reply:
Yes, I checked and this module is only imported from the root `go.mod`, it was created recently with the old Go version